stellargraph/utils/loss.py

```diff
@@ -14,27 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
+import warnings as warnings
```

Comment:
is this just `import warnings`?

Reply:
Oh, yeah. It was previously `import warnings as _warnings` to avoid it being imported for something like `from stellargraph.utils.loss import *`, but I decided that wasn't worth it, apparently without actually doing the full simplification:

```suggestion
import warnings
```